tpot/base.py

```diff
@@ -190,6 +193,17 @@ def __init__(self, generations=100, population_size=100, offspring_size=None,
         warm_start: bool, optional (default: False)
             Flag indicating whether the TPOT instance will reuse the population from
             previous calls to fit().
+        memory: a Memory object or string, optional (default: None)
+            String 'auto':
```

Comment:
Add a line explaining what the `memory` parameter is here. Make it clear that this is pipeline caching.